kitchen.yml

```diff
@@ -30,16 +30,13 @@ provisioner:
         jenkins_args: '--httpKeepAliveTimeout=300000'
 
 platforms:
-  - name: amazonlinux
-    driver_config:
-      box: mvbcoding/awslinux
   - name: amazonlinux-2
-  - name: centos-6
   - name: centos-7
+  - name: centos-8
   - name: debian-9
   - name: debian-10
-  - name: ubuntu-16.04
   - name: ubuntu-18.04
+  - name: ubuntu-20.04
   - name: windows-2012r2
```

Comment:
Please remove, 2012 is old.

Reply:
FWIW I couldn't figure out the actual windows support in this cookbook. If you want, I can dive into it more but it might be better in another PR. I mainly wanted to get this shipped out in a mostly workable format.

Reply:
Let's make it another PR :)